### ISSUE TITLE
RigidBody bounce fails to come to rest.

### ISSUE THREAD

**Issue by avencherus** [godotengine/godot#24332]
Win10 64bit d030c17

![never settling](https://user-images.githubusercontent.com/13004169/49969120-69013500-ff30-11e8-9094-68d7d217dcfe.gif)

In modifying the properties of a RigidBody2D to get a desirable effect, I found that when using higher gravity scales and mass, bodies with bouncing tend to never come to rest.  At higher values they will jitter on the ground.

Absorption can stop it, but it also kills the bounce effect.

Possible I'm misusing something, but it is at least odd to get these results with very little modification.

Demo Project: [3.1 Jittering Bounce.zip](https://github.com/godotengine/godot/files/2677817/3.1.Jittering.Bounce.zip)

**Comment by avencherus** [comment]
Closing, this is the same as #25186

**Comment by hirsimaki-markus** [comment]
I have found a somewhat reliable workaround for this. The ball in below game can only jump if its colliding e.g. not agitated.

![quantum-yeet](https://user-images.githubusercontent.com/46528769/78435132-ea795b00-767e-11ea-957f-d6caabf893eb.gif)

I confirmed it to work with high weights like 100. Values like weight and bounciness might require tweaking along with the threshold below but this should stop infinite bouncing:

```gdscript
func _physics_process(delta):
	if abs(linear_velocity.y) <= 100:
		physics_material_override.absorbent = 1
	else:
		physics_material_override.absorbent = 0
```

I have no idea how to contribute to the source but I suppose the above logic could be implemented directly into source.
